testing/gnome-online-accounts/APKBUILD

```
@@ -1,40 +1,37 @@
-# Maintainer: 
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
 pkgname=gnome-online-accounts
-pkgver=3.26.0
-pkgrel=1
-pkgdesc="client for GNOME web services"
-url="https://www.gnome.org/"
+pkgver=3.32.0
+pkgrel=0
+pkgdesc="Single sign-on framework for GNOME"
+url="https://wiki.gnome.org/Projects/GnomeOnlineAccounts"
 # limited by webkit2gtk
 arch="x86 x86_64 ppc64le aarch64"
```

Can this package be built on all architectures?

```suggestion
arch="all"
```

---

Will try, thanks for the heads-up